src/hotspot/share/runtime/globals.hpp

```diff
@@ -1351,8 +1351,10 @@ const int ObjectAlignmentInBytes = 8;
           "Number of receiver types to record in call/cast profile")        \
           range(0, 8)                                                       \
                                                                             \
-  develop(intx, BciProfileWidth,      2,                                    \
+  develop(intx, BciProfileWidth, 2,                                         \
```

Comment:
Recently, I've seen someone complaining about useless use of `intx`, saying that is brings less readability than a more fixed-width type when not needed. Here, [0, 5000] fits in 16 bits (even signed). One could change that into a simple `int` or something like that.